wpcli-lbf-setup

```diff
@@ -103,8 +103,14 @@ siteurl=`wp option get siteurl 2> /dev/null`
 
 if [ $? -ne 0 ]
 then
-  echo "${red}Could not connect, please re-check your database settings.${normal}"
+  echo "${red}Could not connect, please re-check your database settings. Also check you're in the correct directory.${normal}"
   echo
+  echo "${yellow}Removing..${normal} ${yml_path}"
```

Comment:
I appreciate the thought here and I understand that in many cases you may want to delete these files, however I can imagine some scenarios where removing the files would be unhelpful.

For example, sometimes I'll try to create a connection and get something like the `port` incorrect. In this case, it is easier for me to modify the files than to have them removed.

I think leaving the files is a good start if someone is debugging a problem.

Would you please remove these additions from this PR?

I can imagine having a separate task to delete the files could be helpful, e.g.

```
wpcli-lbf-setup --remove-files
```

If you're interested in tackling something like this, I'd ask that you open an Issue describing the idea and then make a separate PR to add this functionality.

Reply:
I added two commits to change the wording of the message and to remove the file removing. 

I will definitely consider adding that option for file removal! Thank you for taking the time to review my PR.